docs/snippets/quickstart-install.md

```diff
@@ -36,15 +36,15 @@
              kn quickstart --help
              ```
 
-The `quickstart` plugin completes the following functions:
+The `kn-quickstart` plugin completes the following functions:
 
 1. **Checks if you have [`kind`](https://kind.sigs.k8s.io/docs/user/quick-start){target=_blank} installed,** and creates a cluster called `knative`.
 1. **Installs Knative Serving with Kourier** as the default networking layer, and nip.io as the DNS.
 1. **Installs Knative Eventing** and creates an in-memory Broker and Channel implementation.
 
-!!! todo "Install Knative and Kubernetes on a local Docker Daemon using `kn quickstart`"
+!!! todo "Install Knative and Kubernetes on a local Docker Daemon using `kn-quickstart`"
```

Comment:
The intention of the plugin is to be use by the kn CLI and not directly the kn-quickstart binary. We want user to place kn-quickstart binary in the PATH and use "kn quickStart"